### ISSUE TITLE
Labelme Carsh

### ISSUE THREAD

**Issue by LLL151** [wkentaro/labelme#1353]
### Provide environment information

Windows 11 22H2
Labelme 5.3.1

### What OS are you using?

Windows 11 22H2

### Describe the Bug

When the image file is on SSD,software crash occurs when saving JSON files.If using a mechanical hard drive,there will be no problem.This situation has been replicated on four computers,if you use old version Labelme also crash.

SSD (C:)
Mechanical Hard Drive (D:)

### Expected Behavior

No Problem

### To Reproduce

1.Open Dir
2.Create Polygons
3.Save
4.Software Crash

**Comment by augusteastoncalabria** [comment]
This seems like the same behavior I described in #1348
What error do you see upon crashing?

**Comment by wkentaro** [comment]
> *This was generated by AI during triage.*

Re-triaged following @augusteastoncalabria's 2024-01-29 comment.

## Triage Notes

**What we've established so far:**

- The question asked in this thread ("what error do you see upon crashing?") was never answered, and that is still the blocker: there is no traceback, no console output, and no sample file here, so there is nothing to diagnose against the code.
- This is **not** being treated as a duplicate of #1348. That issue has a concrete traceback (`TypeError` from a PyQt5 signature mismatch inside the label-list paint delegate, on Raspbian) which is mechanistically unrelated to a disk-speed-dependent crash on Windows. "Crashes right after saving" is too generic a similarity to merge on.
- The SSD-vs-HDD theory is unlikely to be the real variable for a JSON write. More probable is that the two folders differ in something else (image format, dimensions, file size, path characters, permissions), which is why a sample file matters.
- Two things have changed since this was filed against 5.3.1 that make a retest worthwhile: labelme now writes unhandled exceptions to a log file, and it migrated from PyQt5 to PySide6, which retires a whole class of binding-related crash.

**What we still need from you (@LLL151):**

- Does this still happen on the current release? Please `pip install -U labelme` and retry the same folder on the SSD.
- If it still crashes, run `labelme` from PowerShell rather than double-clicking, and paste everything printed to the console at the moment it crashes.
- After a crash, paste the last ~50 lines of `%LOCALAPPDATA%\labelme\labelme.log`. This captures the traceback even when the window dies.
- What exactly triggers it: the Save button, `Ctrl+S`, or auto-save? Does it crash on the very first shape, or only after several?
- Can you attach one image from the SSD folder that reliably reproduces it (or, if you cannot share it, its format, pixel dimensions, and file size)?
